### DISCUSSION TITLE
A Call for Writing article page/feature

### DISCUSSION THREAD

**Discussion by abderrahmaneMustapha** [dzcode-io/dzcode.io#392] [Ideas]
I have an idea about making people contribute to dzcode, a call for Writing article just loke those Scientifique journals and popular blogs, for example, we write the idea and the title of the blog post share it with others and see if someone is interested and can write a post because most of the time people can and want to write, but they don't have ideas.

what do you think @dzcode-io/article-writers  @dzcode-io/maintainers  @dzcode-io/members ?

**Comment by souhaib-benbouzid** [comment]
I think it's a good idea for a starter to generate some content for the community. but the aim should be for the users to come up with their own topics.

**Comment by omdxp** [comment]
So @abderrahmaneMustapha you want to make the user some suggestions when posting any articles? If so, based on what those suggestions will be ?
